### PR TITLE
[Maintenance] Update ConfirmCategoryDelete, GoCardlessExternalMsg, ManageRulesModal to tsx

### PR DESCRIPTION
**Pull request by MikesGlitch**
Update ConfirmCategoryDelete, GoCardlessExternalMsg, ManageRulesModal to tsx

https://github.com/actualbudget/actual/issues/1483

### CODE REVIEW

File: packages/desktop-client/src/components/modals/GoCardlessExternalMsg.tsx [hunk 1]
```diff
@@ -140,7 +155,6 @@ export default function GoCardlessExternalMsg({
           <Autocomplete
             strict
             highlightFirst
-            disabled={isConfigurationLoading}
```
Comment (MikesGlitch) [inline packages/desktop-client/src/components/modals/GoCardlessExternalMsg.tsx:143]: Not a valid prop on autocomplete. If we want to disable it we set the "inputProps" prop

File: packages/desktop-client/src/components/modals/ManageRulesModal.tsx [hunk 2]
```diff
@@ -16,14 +25,14 @@ export default function ManageRulesModal({ modalProps, payeeId }) {
...
   }
+  debugger;
   return (
     <Modal
       title="Rules"
       padding={0}
       loading={loading}
       {...modalProps}
       style={{
-        ...modalProps.style,
```
Comment (MikesGlitch) [inline packages/desktop-client/src/components/modals/ManageRulesModal.tsx:26]: never set, unused

**Review by MatissJanis — APPROVED** [review]
Great as always!

**Comment by MatissJanis** [comment]
Safe to merge so breaking merge-freeze here. Only type changes that do no affect customer experience. And removal of two things that have very good justification (also double checked myself).

SAFE

**Pull request merged**